linera-ethereum/src/test_utils/mod.rs

```diff
@@ -14,7 +14,7 @@ use linera_alloy::{
     sol,
     transports::http::reqwest::Client,
 };
-use linera_storage_service::child::get_free_port;
```

Comment:
We can argue about whether it should be in `linera-base` or in another crate.
However, to leave it in `linera_storage_service` does not make much sense. It makes a dependency to something that is not designed for that.

And I do not think this characterization of `linera-base` is correct because `mod command` is part of it.

Reply:
It's what [`linera-base`'s crate documentation](https://docs.rs/linera-base/latest/linera_base/) says. I agree that `command` should also be moved elsewhere.

(Anyway, not a blocker for this PR.)